apps/backend/prisma/schema.prisma

```diff
@@ -63,313 +74,230 @@ model Entite {
-  id    String @id @default(uuid())
-  label String @unique
-
-  victimes   Victime[]
-  declarants Declarant[]
-}
-
-model LienVictimeEnum {
-  id         String      @id @default(uuid())
-  label      String      @unique
-  declarants Declarant[]
-}
-
-model Adresse {
-  id         String  @id @default(uuid())
-  label      String?
-  numero     String?
-  rue        String?
-  codePostal String?
-  ville      String?
+  lienVictimeId String?
+  lienVictime   LienVictimeEnum? @relation(fields: [lienVictimeId], references: [id])
 
-  victimes      Victime[]
-  declarants    Declarant[]
-  lieuxIncident LieuIncident[]
-}
+  declarantDeId String?        @unique
+  declarantDe   RequeteEntite? @relation("DeclarantRequeteEntite", fields: [declarantDeId], references: [id], onDelete: Cascade)
 
-model DescriptionFaits {
-  estMaltraitance Boolean?
-  dateDebut       DateTime?
-  dateFin         DateTime?
-  commentaire     String?
+  requeteEntiteId String
+  requete         RequeteEntite @relation("ParticipantsRequeteEntite", fields: [requeteEntiteId], references: [id], onDelete: Cascade)
 
-  requeteEntiteStateId String                             @id
-  requeteEntiteState   RequeteState                       @relation(fields: [requeteEntiteStateId], references: [id], onDelete: Cascade)
-  motifs               DescriptionFaitsMotif[]
-  consequences         DescriptionFaitsConsequence[]
-  maltraitanceTypes    DescriptionFaitsMaltraitanceType[]
+  createdAt DateTime @default(now())
+  updatedAt DateTime @updatedAt
 }
 
-model MotifEnum {
+model LienVictimeEnum {
   id    String @id @default(uuid())
   label String @unique
 
-  descriptionFaits DescriptionFaitsMotif[]
+  PersonneConcernees PersonneConcernee[]
 }
 
-model ConsequenceEnum {
-  id    String @id @default(uuid())
+model AgeEnum {
+  id    String @id
   label String @unique
 
-  descriptionFaits DescriptionFaitsConsequence[]
+  PersonneConcernees PersonneConcernee[]
 }
 
-model MaltraitanceTypeEnum {
-  id    String @id @default(uuid())
+model CiviliteEnum {
+  id    String @id
   label String @unique
 
-  descriptionFaits DescriptionFaitsMaltraitanceType[]
+  personneConcernee PersonneConcernee[]
 }
 
-model DescriptionFaitsMotif {
-  faitsId          String
-  descriptionFaits DescriptionFaits @relation(fields: [faitsId], references: [requeteEntiteStateId], onDelete: Cascade)
+model Identite {
+  id          String @id @default(uuid())
+  prenom      String @default("")
+  nom         String @default("")
+  email       String @default("")
+  telephone   String @default("")
+  commentaire String @default("")
 
-  motifId String
-  motif   MotifEnum @relation(fields: [motifId], references: [id])
+  personneConcerneeId String            @unique
+  personneConcernee   PersonneConcernee @relation("PersonneIdentite", fields: [personneConcerneeId], references: [id], onDelete: Cascade)
 
-  @@id([faitsId, motifId])
+  createdAt DateTime @default(now())
+  updatedAt DateTime @updatedAt
```

Comment:
Ici et les autres, j'aurais mis également en français

Reply:
Invalid character -> everything that does not match the regex _a-zA-Z0-9
Invalid at start of identifier -> everything that does not match the regex a-zA-Z